x-pack/plugins/siem/public/components/page/hosts/first_last_seen/index.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLoadingSpinner } from '@elastic/eui';
+import moment from 'moment';
+import React from 'react';
+import { ApolloConsumer } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { useHostLastFirstSeenQuery } from '../../../../containers/hosts/last_first_seen';
+import { getEmptyTagValue } from '../../../empty_value';
+import { PreferenceFormattedDate } from '../../../formatted_date';
+import { LocalizedDateTooltip } from '../../../localized_date_tooltip';
+
+export type FirstLastSeenType = 'first-seen' | 'last-seen';
+
+export const FirstLastSeen = pure<{ hostname: string; type: FirstLastSeenType }>(
```

Comment:
Do we want to align on naming between `FirstLastSeen` and `LastFirstSeen`. Seems the `server` and `container` names are `LastFirstSeen`, but the component itself is `FirstLastSeen`.  `GlobalTimeBoundaries` or something similar might be a suitable abstraction as well?

Reply:
let me rename it `FirstLastSeenHost` everywhere